### PR TITLE
grammar-graphics-typo

### PR DESCRIPTION
**Pull request by theyoude**
sorry didn't know how to properly edit simply the article, hope this is fine...

### CODE REVIEW

File: content/blog/2018-09-01-grammar-of-graphics.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ In the real world, we usually don't communicate with the simplest kind of senten
 
 #### Facets
 
-Facetting a graphic means to plot small different subsets of the data. Relating to grammar, I would compare the action of facetting with the idea of breaking a long sentence in some shorter (clearer) sentences. It's a powerfull tool when looking for some patterns in the data as it could help removing the noise of differents group plotted all togheter.
+Facetting a graphic means to plot small different subsets of the data. Relating to grammar, I would compare the action of facetting with the idea of breaking a long sentence in some shorter (clearer) sentences. It's a powerful tool when looking for some patterns in the data as it could help removing the noise of different groups plotted all togheter.
 
 For example, it's difficult to see the patterns inside each group in the [figure below](#fig:facet1):
```
Comment (samperochkin) [inline content/blog/2018-09-01-grammar-of-graphics.md]:
```suggestion
For example, it's difficult to see the true pattern of each group in the [following graphic](#fig:facet1):
```

File: content/blog/2018-09-01-grammar-of-graphics.md [hunk 2]
```diff
@@ -226,17 +223,17 @@ This part is probably the toughest to learn and certainly needs a lot of practic
...
+This is probably the most important thing to keep in mind while developping a graphical visualization. It should always guide our decisions (type of graphic, theme, aesthetics, etc.) because at the end, the graphic should help people understand something specific.
 
 ## Data
 
 The data is one of the first thing to take into account before working on a graphic. The type of data, the size of the data and so on should by itself indicate what is doable and what is not or what transformations should be done.
 
 ## Our audience
 
-The audience is always an important aspect of every communications. It should be taken into account and should also guide our decisions regarding our graphic. For example, boxplots can be quite meaningful for a given audience such as data scientists or statisticians but mean nothing to someone else.
+The audience is always an important aspect of every communication. It should be taken into account and should also guide our decisions regarding our graphic. For example, boxplots can be quite meaningful for a given audience such as data scientists or statisticians but mean nothing to someone else.
```
[the remark this answers is not in the document]
Comment (theyoude) [reply content/blog/2018-09-01-grammar-of-graphics.md]: you are right, communication is for a specific exchange while communicationS means a system for transmitting information, so the S can be kept

File: content/blog/2018-09-01-grammar-of-graphics.md [hunk 3]
```diff
@@ -125,11 +125,8 @@ ggplot(mtcars, aes(x = hp, y = mpg, color = as.factor(carb))) +
 
 <div id="fig:facet1" class="figure" style="text-align: center">
 <img src="2018-09-01-grammar-of-graphics_files/figure-html/facet1-1.png" alt="It's difficult to see the true pattern in each group here"  />
-<p class="caption">It's difficult to see the true pattern in each group here</p>
-</div>
-
-while it's quite clearer in the [figure below](#fig:facet2).
 
+It's difficult to see the true pattern of each group in the previous graphic while it's quite clearer in the [figure below](#fig:facet2).
```
Comment (samperochkin) [inline content/blog/2018-09-01-grammar-of-graphics.md]:
```suggestion
while it's quite clearer in the [figure below](#fig:facet2).
```

File: content/blog/2018-09-01-grammar-of-graphics.md [hunk 4]
```diff
@@ -125,11 +125,8 @@ ggplot(mtcars, aes(x = hp, y = mpg, color = as.factor(carb))) +
 
 <div id="fig:facet1" class="figure" style="text-align: center">
 <img src="2018-09-01-grammar-of-graphics_files/figure-html/facet1-1.png" alt="It's difficult to see the true pattern in each group here"  />
-<p class="caption">It's difficult to see the true pattern in each group here</p>
-</div>
-
-while it's quite clearer in the [figure below](#fig:facet2).
```
Comment (samperochkin) [inline content/blog/2018-09-01-grammar-of-graphics.md]:
```suggestion
<p class="caption">It's difficult to see the true pattern of each group here</p>
</div>

```

**Pull request merged**